.github/workflows/publish.yml

```diff
@@ -19,18 +19,30 @@ jobs:
           username: ${{ secrets.DOCKER_LOGIN_USERNAME }}
           password: ${{ secrets.DOCKER_LOGIN_PASSWORD }}
       
-      - name: Extract metadata (tags, labels) for Docker
-        id: meta
+      - name: Extract intelpython3_core metadata (tags, labels) for Docker
+        id: meta_core
         uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
         with:
-          images:
-            - intelpython/intelpython3_core
-            - intelpython/intelpython3_full
+          images: intelpython/intelpython3_core
       
-      - name: Build and push Docker image
+      - name: Extract intelpython3_full metadata (tags, labels) for Docker
+        id: meta_full
+        uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+        with:
+          images: intelpython/intelpython3_full
+
+      - name: Build and push Docker image for intelpython3_core
+        uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
+        with:
+          context: .
+          push: true
+          tags: ${{ steps.meta_core.outputs.tags }}
+          labels: ${{ steps.meta_core.outputs.labels }}
+
+      - name: Build and push Docker image for intelpython3_full
         uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
         with:
           context: .
```

Comment:
This should be:

```
context: configs/intelpython3_full
```
and similar for python3_core

Reply:
Hmm, I see that CI failed with this change :( https://github.com/IntelPython/container-images/pull/29